tests/ctst/world/Zenko.ts

```diff
@@ -52,44 +83,41 @@ export default class Zenko extends World {
 
     private forceFailed = false;
 
+    readonly parameters: Record<string, unknown> = {};
+
     private cliMode: cliModeObject = CacheHelper.createCliModeObject();
 
     /**
      * @constructor
      * @param {Object} options - parameters provided as a CLI parameter when running the tests
      */
-    constructor(options: any) {
+    constructor(options: IWorldOptions) {
         super(options);
 
         // store service users credentials from world parameters
         if (this.parameters.ServiceUsersCredentials) {
-             Object.entries(JSON.parse(this.parameters.ServiceUsersCredentials))
-                .forEach((entry : [string, any]) => {
-                    Zenko.serviceUsersCredentials[entry[0] as string] = {
-                        AccessKeyId: entry[1].accessKey as string,
-                        SecretAccessKey: entry[1].secretKey as string,
-
+            Object.entries(JSON.parse(
+                this.parameters.ServiceUsersCredentials as string) as UserCredentials)
+                .forEach(entry => {
+                    Zenko.serviceUsersCredentials[entry[0]] = {
+                        AccessKeyId: (entry[1] as { accessKey: string }).accessKey,
+                        SecretAccessKey: (entry[1] as { secretKey: string }).secretKey,
                     };
                 });
         }
 
         // Workaround to be able to access global parameters in BeforeAll/AfterAll hooks
         CacheHelper.parameters = this.parameters;
-        this.cliMode.parameters = this.parameters;
+        this.cliMode.parameters = this.parameters as ClientOptions;
 
         if (this.parameters.AccountSessionToken) {
-            CacheHelper.ARWWI[CacheHelper.AccountName] = {
+            ((CacheHelper.ARWWI as Record<string, unknown>)[CacheHelper.AccountName]) = {
                 AccessKeyId: this.parameters.AccountAccessKey,
                 SecretAccessKey: this.parameters.AccountSecretKey,
                 SessionToken: this.parameters.AccountSessionToken,
             };
-            CacheHelper.parameters.AccessKey = this.parameters.AccountAccessKey;
-            CacheHelper.parameters.SecretKey = this.parameters.AccountSecretKey;
-            CacheHelper.parameters.SessionToken = this.parameters.AccountSessionToken;
         } else {
-            CacheHelper.AccountName = CacheHelper.parameters.AccountName;
-            CacheHelper.parameters.AccessKey = this.parameters.AccountAccessKey;
-            CacheHelper.parameters.SecretKey = this.parameters.AccountSecretKey;
+            CacheHelper.AccountName = this.parameters.AccountName as string;
```

Comment:
I would argue that all the parameters should be already known, as we handle them with our own logic, so the parameters type should be properly typed, to avoid `unknwon` that leads to many `as` in the code logic, making it hard to maintain.

Reply:
Done when adding World parameters from cucumber bump